test/integration/get_statvar_path_test.go

```diff
@@ -30,28 +33,21 @@ func TestGetStatVarPath(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to set up mixer and client")
 	}
+	_, filename, _, _ := runtime.Caller(0)
+	goldenPath := path.Join(
+		path.Dir(filename), "golden_response/get_statvar_path")
 
 	for _, c := range []struct {
-		id   string
-		want []string
+		id         string
+		goldenFile string
 	}{
 		{
 			"Count_Person",
-			[]string{
-				"Count_Person",
-				"dc/g/Variables_Demographics",
-				"dc/g/Demographics",
-			},
+			"person.json",
 		},
 		{
-			"dc/g/Person_Age_ArmedForcesStatus_Employment_EmploymentStatus",
-			[]string{
-				"dc/g/Person_Age_ArmedForcesStatus_Employment_EmploymentStatus",
-				"dc/g/Person_Age_Employment_EmploymentStatus",
-				"dc/g/Person_Age_Employment",
-				"dc/g/Person_Age",
-				"dc/g/Demographics",
-			},
+			"dc/g/Person_DetailedLevelOfSchool-EnrolledInGrade10",
+			"school.json",
```

Comment:
what is the plan here, are we going to promote this to PROD?

Reply:
Once the website change is also in, we can do some thorough testing. This could be candidate for next week's prod release.

Reply:
Probably want a newer PV candidate (built on Monday), so it includes BlockGroups. But the plan SG.